client/src/App.js

```diff
@@ -13,6 +14,7 @@ function App() {
       <Route path="/users/signup" element={<SignUp />} />
```

Comment:
Add `path='*'` and create a `404` component.

Reply:
This will be taken care in the very next commit for sure.